### DISCUSSION TITLE
怎么在删除hc:Tag时 给Closing 事件 绑定命令

### DISCUSSION THREAD

**Discussion by simadi** [HandyOrg/HandyControl#1360] [Q&A] [answered]
<hc:TagContainer Grid.Row="1" ItemContainerStyle="{StaticResource TagCustomStyle}" Margin="11" Padding="5" ItemsSource="{Binding DataList}"/>

        <Style x:Key="TagCustomStyle" BasedOn="{StaticResource TagBaseStyle}" TargetType="hc:Tag">
            <Setter Property="Margin" Value="5"/>
            <Setter Property="FontSize" Value="14"/>
            <Setter Property="Content" Value="{Binding RoleName}"/>
            <Setter Property="ShowCloseButton" Value="{Binding CanDele}"/>
            <Style.Triggers>
                <EventTrigger RoutedEvent="Closing">
                    这里要绑定一个命令,对应的命令要删除库里的数据,该怎么绑定这个命令??求指教谢谢
                </EventTrigger>
            </Style.Triggers>
        </Style>

**Comment by NaBian** [comment, marked as answer]
``` xaml
<hc:TagContainer Tag.Closing="事件响应方法" Grid.Row="1" ItemContainerStyle="{StaticResource TagCustomStyle}" Margin="11" Padding="5" ItemsSource="{Binding DataList}"/>
```

**Comment by simadi** [comment]
thank u!
